snippets/product/type/with-variations/product.liquid

```diff
@@ -0,0 +1,72 @@
+<div class="row">
+    <div class="col-xs-12">
+        <!-- <a href="#">{{ product.properties['Brand'].value }}</a> -->
+        <h1>{{ product.title }}</h1>
+        <ul class="list-inline">
+            <li>
+                <span class="text-muted">{{ 'products.product.sku' | t }}</span>&nbsp;<strong itemprop="productID">{{ product.sku }}</strong>
+            </li>
+            <li>
+                <span class="text-muted">{{ 'products.product.mmn' | t }}</span>&nbsp;<strong itemprop="model">{{ product.properties['MMN'].value }}</strong>
+            </li>
+            <li>
+                <span class="text-muted">UNSPSC</span>&nbsp;<strong>#41112701</strong>
+            </li>
+        </ul>
+    </div>
+</div>
```

Comment:
This part looks exactly the same as for other types. May we reuse the code?

Reply:
done